testing/test-pngs.py

```
@@ -0,0 +1,39 @@
+from http.server import HTTPServer, SimpleHTTPRequestHandler
+import os
```

Ruff

`os` imported but unused

Remove unused import: `os`

(F401)